docs/compute/persistent_tpm_and_uefi_state.md

```diff
@@ -8,8 +8,10 @@ As of v1.0.0, KubeVirt supports using a PVC to persist those files. KubeVirt usu
 
 ## Backend storage
 
-KubeVirt automatically creates backend storage PVCs for VMs that need it. However, the admin must first enable the `VMPersistentState` feature gate, and tell KubeVirt which storage class to use by setting the `vmStateStorageClass` configuration parameter in the KubeVirt Custom Resource (CR).  
-The storage class must support read-write-many (RWX) in filesystem mode (FS).  
+KubeVirt automatically creates backend storage PVCs for VMs that need it. However, to persist TPM and UEFI state, the admin must first enable the `VMPersistentState` feature gate.
+
+If [live migration](live_migration.md) is required, then the `vmStateStorageClass` configuration parameter should be set, and must reference a storage class that supports read-write-many (RWX) in filesystem mode (FS).
```

Comment:
Not anymore. In 1.4, RWO FS backend-storage is migratable. `vmStateStorageClass` can still be used to manually specify a storage class though, instead of relying on defaults.

Reply:
I have pushed an update to address this.